### PR TITLE
Add new methods to documentation

### LINKED ISSUE #10: docs missing info
where is GET for ?=date=2017-09-30
where is POST one point documentation

### PR DESCRIPTION
**Pull request by demyanenko**
Closes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,4 +54,6 @@ Examples of the points (rename `tags.json.example` to `tags.json` for successful
 ## Methods
 * POST `/api/points` with point in the body adds a new point to the storage, validating it against corresponding tag
 * GET `/api/points` returns the entire storage contents
-* PUT `/api/points` overwrites the entire storage, the body format is the same as result of the previous method; all points are validated against tags, if any single one doesn't pass validation, storage is not modified
\ No newline at end of file
+* GET `/api/points?startDate=yyyy-mm-dd` returns all points that were added on or after specified date
+* PUT `/api/points` overwrites the entire storage, the body format is the same as result of the previous method; all points are validated against tags, if any single one doesn't pass validation, storage is not modified
```
Comment (Emettant) [inline README.md]: I would add smth like "Use with caution next api calls" or "build client with confirmation window only"

File: README.md [hunk 2]
```diff
@@ -54,4 +54,6 @@ Examples of the points (rename `tags.json.example` to `tags.json` for successful
 ## Methods
 * POST `/api/points` with point in the body adds a new point to the storage, validating it against corresponding tag
 * GET `/api/points` returns the entire storage contents
-* PUT `/api/points` overwrites the entire storage, the body format is the same as result of the previous method; all points are validated against tags, if any single one doesn't pass validation, storage is not modified
\ No newline at end of file
+* GET `/api/points?startDate=yyyy-mm-dd` returns all points that were added on or after specified date
```
Comment (Emettant) [inline README.md]: nit:
"added on or after"?
returns all points that were added on < yyyy-mm-dd > or after < yyyy-mm-dd >

Comment (demyanenko) [inline README.md]: The wording is grammatically correct.

File: README.md (hunk 1 above)
Comment (demyanenko) [inline README.md]: In my opinion the words "overwrites the entire storage" fully describe the danger of using this method. This is a basic action on backend API, it doesn't require warnings.

**Comment by Emettant** [comment]
molodec

**Pull request merged**